apps/extension/package.json

```
@@ -22,6 +24,7 @@
     "@eslint/js": "^9.29.0",
     "@pinback/eslint": "workspace:*",
     "@pinback/typescript": "workspace:*",
+    "@pivanov/vite-plugin-svg-sprite": "^3.0.0",
```

Biome (1.9.4)

[error] The key @pivanov/vite-plugin-svg-sprite was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)